src/fairseq2/models/wav2vec2/asr/factory.py

```diff
@@ -39,6 +39,30 @@ def _base_100h_encoder() -> Wav2Vec2EncoderConfig:
     return config
 
 
+def _ls960_encoder() -> Wav2Vec2EncoderConfig:
+    config = wav2vec2_encoder_archs.get("large_ls960")
+
+    config.feature_gradient_scale = 1.0
+    config.dropout_p = 0.0
+    config.attn_dropout_p = 0.0
+    config.ffn_inner_dropout_p = 0.1
+    config.layer_drop_p = 0.1
+
+    return config
+
+
+def _lv60k_encoder() -> Wav2Vec2EncoderConfig:
```

Comment:
And this `_large_lv60k_encoder`, again to be consistent with the pretrained `large_lv60k` name.